.github/workflows/ci.yml

```diff
@@ -83,6 +84,10 @@ jobs:
           wget -qO- get.nextflow.io | bash
           sudo mv nextflow /usr/local/bin/
 
+      - name: Download GTF for additional annotation
+        run: |
+          wget https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/009/858/895/GCA_009858895.3_ASM985889v3/GCA_009858895.3_ASM985889v3_genomic.gtf.gz
+
```

Comment:
If I'm not misunderstanding, you're downloading this every time in this matrix.
Can you add an if statement on the params for this step?

Reply:
Sure! I also added it in nanopore when it downloads the medaka model. Thanks for the suggestion!